packager/packager.cc

```diff
@@ -1231,6 +1259,27 @@ struct PackageSpecBodyWalk {
 
         illegalNode(ctx, original);
     }
+
+private:
+    bool isValidStrictDependenciesOption(core::NameRef value) {
+        return value == core::Names::false_() || value == core::Names::layered() ||
+               value == core::Names::layeredDag() || value == core::Names::dag();
+    }
+
+    core::packages::StrictDependenciesLevel optionToEnum(core::NameRef value) {
```

Comment:
Feedback that should be obsoleted by my other suggestion to parse instead of validate, but "optionToEnum" is a very broad name—it does not capture that this method's implementation is for the strict dependencies option only.